REINDEXING.md

```diff
@@ -0,0 +1,261 @@
+# Reindexing
+
+A reindex operation runs the source data from the [adapters](docs/adapters/README.md) through the pipeline causing it to be re-transformed / matched & merged as appropriate.
+
+## How to run a reindex
+
+Running a reindex will require:
+
+- Preparing and deploying a release
+- Terraforming a new pipeline using that release
+- Running the reindex script to feed the new pipeline
+
+### Update the catalogue-pipeline environments
+
+The catalogue-pipeline is deployed using [weco-deploy](https://github.com/wellcomecollection/weco-deploy) but does not deploy in the usual way as there is often one release per pipeline. 
```

Comment:
```suggestion
The catalogue-pipeline is deployed using [weco-deploy](https://github.com/wellcomecollection/weco-deploy) but does not deploy in the usual way as there is often one environment per pipeline. 
```

Reply:
I'm going to omit the bit "but does not deploy in the usual way" is it begs explanation which would be unnecessary detail here. I'm going to state only "Each environment represents a new pipeline."